pythonforandroid/recipes/kivy/__init__.py

```diff
@@ -6,8 +6,7 @@
 
 
 class KivyRecipe(CythonRecipe):
-    # post kivy==1.10.1 with SDL 2.0.9 fixes
-    version = 'b47f669'
+    version = 'master'
```

Comment:
why unpinning rather than linking to a "stable" commit hash e.g. the last one https://github.com/kivy/kivy/commit/775cd9e6e39809ad52de10280a5aa8d76c56a24c ?
```python
version = '775cd9e'
```
I personally prefer reproducible builds, but I know this can be rather controversial

Reply:
:+1: on the pin

Reply:
@AndreMiras and @JonasT, I totally agree with both of you...so ...`pin` it's better...thanks @AndreMiras :smile:

¡¡I will change that!!